### DISCUSSION TITLE
Forward synctex not working in VSCode Insiders

### DISCUSSION THREAD

**Discussion by asapsmc** [James-Yu/LaTeX-Workshop#3146] [Q&A]
I'm using MacOs and VSCode Insiders, as LateX editing becomes excruciatingly slow in VSCode.
I can reverse sync, but I can't do forward sync (from tex to pdf) with cmd+option+j, it just does nothing.

**Comment by tamuratak** [comment]
Please fill out the [issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md) next time.
